app.js

```diff
@@ -1,37 +1,48 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const session = require('express-session');
-const passport = require('passport');
-const mongoose = require('mongoose');
-const path = require('path');
-const engine = require('ejs-locals');
+const express = require("express");
+const bodyParser = require("body-parser");
+const session = require("express-session");
+const passport = require("passport");
+const mongoose = require("mongoose");
+const path = require("path");
+const engine = require("ejs-locals");
+const config = require("./config/config.example");
```

Comment:
please read the instructions properly. u need not include config.example, it was clearly mentioned in the readme to just copy it.